spec/models/pacbio/run_spec.rb

```diff
@@ -58,16 +58,17 @@
   end
 
   context 'associations' do
-    it 'can have a plate' do
-      plate = create(:pacbio_plate)
-      run = create(:pacbio_run, plate:)
-      expect(run.plate).to eq(plate)
+    it 'can have multiple plates for Revio' do
+      plate_1 = create(:pacbio_plate, wells: [build(:pacbio_well, row: 'A', column: '1')])
+      plate_2 = create(:pacbio_plate, wells: [build(:pacbio_well, row: 'A', column: '1')])
+      run = create(:pacbio_run, system_name: 'Revio', plates: [plate_1, plate_2])
+      expect(run.plates).to eq([plate_1, plate_2])
     end
 
     it 'can have some wells' do
       wells = create_list(:pacbio_well, 5)
       plate = create(:pacbio_plate, wells:)
-      run = create(:pacbio_run, plate:)
+      run = create(:pacbio_run, plates: [plate])
       expect(run.wells.count).to eq(5)
```

Comment:
Maybe add a second plate to the run, to check how `run.wells.count` is handled when there are multiple plates with multiple wells